Include/internal/pycore_ceval.h

```diff
@@ -193,9 +193,18 @@ extern void _PyEval_DeactivateOpCache(void);
 
 /* --- _Py_EnterRecursiveCall() ----------------------------------------- */
 
-static inline int _Py_MakeRecCheck(PyThreadState *tstate)  {
+static inline uintptr_t
+_Py_get_machine_stack_pointer(void) {
+#if defined(__GNUC__) || defined(__clang__)
```

Comment:
I think this should work:

```suggestion
#if _Py__has_builtin(__builtin_frame_address)
```